virt/install/uninstalling-virt.adoc

```diff
@@ -30,3 +30,5 @@ You can now delete the `openshift-cnv` namespace.
 ====
 
 include::modules/deleting-a-namespace-using-the-web-console.adoc[leveloffset=+1]
+
+include::modules/virt-deleting-virt-cli.adoc[leveloffset=+1]
```

Comment:
You need to put this module in a separate assembly. It's not clear at all that all those other steps are all required to remove OpenShift virtualization by using the web console and that just these steps are required to remove it by using the CLI.

Reply:
Hi - We currently do not provide any content coverage for uninstalling CNV from the CLI. The content coverage is restricted to uninstalling CNV from the web console as you have noted. However, QE had tried to uninstall CNV from the CLI and ran into a situation where the CSV did not get uninstalled unless it was explicitly removed. 

The ideal scenario would be to provide complete coverage on how to uninstall CNV from the CLI with a new assembly and new modules. However, in negotiations with installation engineers it was agreed that this level of comprehensive coverage is beyond the scope of the 2.4 release, and that it would be addressed in the 2.5 release. Audrey, our content strategist/DPM, Andrew Burden, and Pan Ousley agreed that the scope of this defect and this PR would be limited to providing one module with instructions on how to explicitly remove the CSV.

My general understanding is that when we have an assembly, it should have at least two or more modules associated with it. Is it acceptable to have an assembly with this one module? Do you recommend then that I go ahead and create a new assembly and add this module to that assembly?

Thanks, Kathryn for picking up this PR today! Appreciate it. Look forward to your guidance.

Reply:
I think that you've made this assembly cover two user stories, which goes against the mod doc requirements to have one assembly = one user story. In a perfect world, I would present either CLI or web console instructions, not both, and present whichever method was used by the most users based on research.

In the imperfect world in which we live, I would split the assemblies. I think that a separate assembly that includes just this module and a little glue text would be far preferable because it would be clearer what to do and offer a better user experience.

If Audrey wants them in the same assembly, I'd add clear H2 headings in the assembly for the web console and CLI sections and nest the modules for each method as H3s.

Reply:
The web console assembly was a pre-existing assembly that already covered uninstalling from the web console. I had added one new module to this existing assembly with the objective that in the future the entire assembly would be updated with comprehensive coverage of CLI-based uninstallation of virt. No worries - if we allow one module per assembly, I will create a new assembly and move this new module to that new assembly focused on CLI-based uninstallation of CSV. Thanks for the clarification!

Reply:
That sounds good, Lavanya. My suggestion was going to be to follow Kathryn's recommendation because that seems like the better solution in the long-term. If you feel that doing this now is too much, Lavanya, let me know and we'll see if we can move the defect to another writer. Thanks.

Reply:
@lmandavi - sorry to come in late and be slightly dense, I get what you're getting at, but we need to keep things coherent and re-usable for FCC, which means that we have to be militant about the 1:1 relationship between assemblies and user stories - yes, we'll end up repeating some things, but we're likely talking about two different users, so for ease of maintenance and to lighten the load on the user, the FCC requirements make sense, even if they don't always produce what feels like the most elegant solution

Reply:
@kalexand-rh - Based on your guidance and my understanding, this is the assembly architecture that we are aiming at - please let me know if this is the direction to follow. Thanks!

Uninstalling OpenShift virtualization using the web console (ASSEMBLY)
--Deleting the CNV Operator Deployment custom resource
--Deleting the OpenShift virtualization catalog subscription
--Deleting a namespace using the web console

--Uninstalling OpenShift virtualization using the CLI (ASSEMBlY)
--Deleting OpenShift virtualization